samples/python/llm/requirements.txt

```diff
@@ -0,0 +1,16 @@
+numpy
+openvino>=2023.1.0
+pillow
+torch
+transformers>=4.28.0,<=4.33.3
+tokenizers<0.14.0    # for chatglm-6b
+diffusers
+optimum
+git+https://github.com/huggingface/optimum-intel.git@4b8ed2404abf4408a16d251870a8deb25a4c448b
```

Comment:
@jiwaszki Since the LLM related packages are being developed swiftly and being changed fast, we have to use a specific commit ID or even the HEAD of master to support as many models as possible, is it OK to do in this way?

Reply:
It is absolutely understandable now, resolving it.